Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># See https://help.github.com/articles/ignoring-files/ for more about ignoring files.\n\n# dependencies\n/node_modules\n/.pnp\n.pnp.js\n\n# testing\n/coverage\n\n# production\n/build\n\n# misc\n.DS_Store\n.env.local\n.env.development.local\n.env.test.local\n.env.production.local\n\nnpm-debug.log*\nyarn-debug.log*\nyarn-error.log*\n/.idea/.gitignore\n/.idea/misc.xml\n/.idea/modules.xml\n/.idea/inspectionProfiles/Project_Default.xml\n/src/store/photos/Sittings.PNG\n/src/store/photos/TopScreen.PNG\n/.idea/vcs.xml\n/.idea/.name\n/public/favicon.ico\n/.idea/medias.iml\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 677a525871d8d1fdcb1f7954c3c84f78f3389f5a)
+++ b/.gitignore	(date 1648494212162)
@@ -31,3 +31,4 @@
 /.idea/.name
 /public/favicon.ico
 /.idea/medias.iml
+/package-lock.json
